scipy/stats/tests/test_continuous.py

```diff
@@ -1148,6 +1148,80 @@ def pdf(self, x, a, b):
                 assert_allclose(X.moment(order, kind=kind),
                                 Y.moment(order, kind=kind))
 
+    # pdf and cdf formulas below can warn on boundary of support in some cases.
+    # See https://github.com/scipy/scipy/pull/22560#discussion_r1962763840.
+    @pytest.mark.filterwarnings("ignore::RuntimeWarning")
+    @pytest.mark.parametrize("c", [-1, 0, 1, np.asarray([-2.1, -1., 0., 1., 2.1])])
+    def test_custom_variable_support(self, c):
+        rng = np.random.default_rng(7548723590230982)
+
+        class MyGenExtreme:
```

Comment:
I didn't check the formulas line by line. I figured the fact that the results agree with the existing implementation (without *using* the existing implementation) is sufficient, but LMK if you want me to look more carefully.

Reply:
I think the formulas should be fine. The agreement should be enough.